RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeedsParameters_cff.py

```diff
@@ -34,7 +34,11 @@
     # H/E towers
     hcalTowers = cms.InputTag("towerMaker"),
     hOverEPtMin = cms.double(0.),
-    
+
+    applySigmaIEtaIEtaCut = cms.bool(True),
+    # sigma_ietaieta
+    maxSigmaIEtaIEtaBarrel = cms.double(0.5),
+    maxSigmaIEtaIEtaEndcaps = cms.double(0.5),    
```

Comment:
If so, it should be turned off here and this module should be customized such that it is turned on for HI reconstruction.